panels/site_sections/attractions.py

```diff
@@ -151,7 +151,9 @@ def verify_badge_num(self, session, badge_num, **params):
     @ajax
     def signup_for_event(self, session, id, badge_num='', first_name='',
                          last_name='', email='', zip_code='', **params):
-        if badge_num:
+
+        # Badge number during AT_THE_CON is a hard requirement for Autographs
```

Comment:
Ahhh, good idea.  I originally figured there was no harm in allowing badgeless signups even during the event, but that would potentially be really bad for autographs.